emission/core/wrapper/suggestion_sys.py

```diff
@@ -344,9 +344,9 @@ def zipcode_to_city(zipcode):
 
 def calculate_yelp_server_suggestion_singletrip_nominatim(uuid, tripid):
     user_id = uuid
-    trip_id = bson.objectid.ObjectId(tripid)
+    tripidstr = bson.objectid.ObjectId(tripid)
```

Comment:
wait, the current `tripid` is the string. After you wrap it in `bson.objectid.ObjectId`, you get a tripid object, no?
so bson.objectid.ObjectId() converts str -> id, while your change has it be the other way around...